lib/export-scala-base.js

```diff
@@ -162,6 +162,10 @@ const perBusInterfaceWiring = comp => {
           const busPort = e.name + '0.' + tlName;
           const blackboxPort = lPortMap[busName]; // .physicalPort.name;
           const direction = isInputBusPort ? 'out' : 'in';
+          const blackboxPortDirection = portObj[blackboxPort].wire.direction;
+          if (direction !== blackboxPortDirection) {
```

Comment:
might be wrong about what this is doing, but input bus interface signals should be able to be mapped to both input and output blackbox ports since they can be driven by either? I think we only want to do this check if the bus interface port is an output?

Reply:
I'm not sure that I understand. How could an input bus interface signal be driven by both input and output blackbox ports? At the very least the generated Chisel code won't be happy if the port directions don't match.

Reply:
Since I'd like to get the actual fixes to the bus definitions in sooner, let me back out of the change that adds the direction checking and put that in a separate PR. It's also causing many tests to fail, and I'd like to separately determine whether the assertion is incorrect or whether the schemas are just incorrect.